src/handlers/notify_zulip.rs

```diff
@@ -52,26 +55,40 @@ pub(super) async fn parse_input(
 fn parse_label_change_input(
     event: &IssuesEvent,
     label: Label,
-    config: &NotifyZulipLabelConfig,
+    config: &NotifyZulipTablesConfig,
 ) -> Option<NotifyZulipInput> {
-    if !has_all_required_labels(&event.issue, config) {
-        // Issue misses a required label, ignore this event
+    let mut include_config_names: Vec<String> = vec![];
+
+    for (name, label_config) in &config.subtables {
+        if has_all_required_labels(&event.issue, &label_config) {
```

Comment:
Hmm, I wonder how this works (even though this is pre-existing behavior). When the required label was removed (`IssuesAction::Unlabeled`), then `event.issue` will not contain the required label anymore, so `has_all_required_labels` doesn't trigger, right? So how does the code then send the `message_on_remove` message? :thinking:

Reply:
As I understood, `IssuesAction::Labeled` and `IssuesAction::Unlabeled` is only for the title of the table, such as `beta-nomination`. So when we delete a required label, such as, `T-rustdoc`, no message would be sent. It makes sense to me because it is a *required* label so without it, no message should be sent.

Reply:
Oh, I see, thanks for clarifying! A bit confusing that there are two sets of labels involved, but I guess that's just how it works.